scripts/plot_dataframe.py

```diff
@@ -74,52 +76,70 @@ def load_trades(args: Namespace, pair: str, timerange: TimeRange) -> pd.DataFram
         # must align with columns in backtest.py
         columns = ["pair", "profit", "opents", "closets", "index", "duration",
                    "open_rate", "close_rate", "open_at_end", "sell_reason"]
-        with file.open() as f:
-            data = json.load(f)
-            trades = pd.DataFrame(data, columns=columns)
-        trades = trades.loc[trades["pair"] == pair]
-        if timerange:
-            if timerange.starttype == 'date':
-                trades = trades.loc[trades["opents"] >= timerange.startts]
-            if timerange.stoptype == 'date':
-                trades = trades.loc[trades["opents"] <= timerange.stopts]
-
-        trades['opents'] = pd.to_datetime(trades['opents'],
-                                          unit='s',
-                                          utc=True,
-                                          infer_datetime_format=True)
-        trades['closets'] = pd.to_datetime(trades['closets'],
-                                           unit='s',
-                                           utc=True,
-                                           infer_datetime_format=True)
+        if os.path.exists(file):
+            with file.open() as f:
+                data = json.load(f)
+                trades = pd.DataFrame(data, columns=columns)
+            trades = trades.loc[trades["pair"] == pair]
+            if timerange:
+                if timerange.starttype == 'date':
+                    trades = trades.loc[trades["opents"] >= timerange.startts]
+                if timerange.stoptype == 'date':
+                    trades = trades.loc[trades["opents"] <= timerange.stopts]
+
+            trades['opents'] = pd.to_datetime(
+                                            trades['opents'],
+                                            unit='s',
+                                            utc=True,
+                                            infer_datetime_format=True)
+            trades['closets'] = pd.to_datetime(
+                                            trades['closets'],
+                                            unit='s',
+                                            utc=True,
+                                            infer_datetime_format=True)
+        else:
+            trades = pd.DataFrame([], columns=columns)
+
     return trades
 
 
-def plot_analyzed_dataframe(args: Namespace) -> None:
+def generate_plot_file(fig, pair, tick_interval, is_last) -> None:
     """
-    Calls analyze() and plots the returned dataframe
+    Generate a plot html file from pre populated fig plotly object
     :return: None
     """
+    logger.info('Generate plot file for %s', pair)
+
+    pair_name = pair.replace("/", "_")
+    file_name = 'freqtrade-plot-' + pair_name + '-' + tick_interval + '.html'
+
+    if not os.path.exists('user_data/plots'):
```

Comment:
i would prefer to do everything with Pathlib - this could fail depending on the platform (and will fail if plots is a file)

using `Path('user_data/plots').is_dir()` should work ... 
if not os.path.exists('user_data/plots'):